src/app/app-modules/nurse-doctor/nurse-worklist-tabs/nurse-reffered-worklist/nurse-reffered-worklist.component.html

```
@@ -1,225 +1,63 @@
-          autocomplete="off"
-          placeholder="{{ currentLanguageSet?.common?.inTableSearch }}"
-          name="filterTerm"
-          [(ngModel)]="filterTerm"
-          (keyup)="filterBeneficiaryList(filterTerm)" />
-        <button class="go" (click)="filterBeneficiaryList(filterTerm)">
-          <span class="material-icons">search</span>
-        </button>
-      </div>
-    </div>
-    <div class="col-xs-4 col-sm-6 col-md-8 fit_to_content">
+<app-beneficiary-worklist
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)